### ISSUE TITLE
Crash in AppsFlyerLib

### ISSUE THREAD

**Issue by DagAgren** [AppsFlyerSDK/AppsFlyerFramework#192]
# Report

## SDK Version

6.4.2

## What happened?

After https://github.com/AppsFlyerSDK/AppsFlyerFramework/issues/187 was fixed we are still seeing this crash in AppsFlyerLib:

```
Crashed: com.appsflyer.serial
EXC_BAD_ACCESS KERN_INVALID_ADDRESS 0x0000000b392280d0

Crashed: com.appsflyer.serial
0  libobjc.A.dylib                0x1bb2f0154 objc_msgSend + 20
1  WoltClient Production          0x102efa84c -[AppsFlyerLib getEventParameters:] + 4345882700
2  WoltClient Production          0x102efd620 -[AppsFlyerLib callServerWithEventName:eventValues:options:completion:] + 4345894432
3  libdispatch.dylib              0x1a71b7298 _dispatch_client_callout + 16
4  libdispatch.dylib              0x1a715c028 _dispatch_continuation_pop$VARIANT$mp + 412
5  libdispatch.dylib              0x1a716cd64 _dispatch_source_invoke$VARIANT$mp + 1304
6  libdispatch.dylib              0x1a715f908 _dispatch_lane_serial_drain$VARIANT$mp + 300
7  libdispatch.dylib              0x1a7160518 _dispatch_lane_invoke$VARIANT$mp + 420
8  libdispatch.dylib              0x1a7169fac _dispatch_workloop_worker_thread + 712
9  libsystem_pthread.dylib        0x1efb895bc _pthread_wqthread + 272
10 libsystem_pthread.dylib        0x1efb8c86c start_wqthread + 8
```

## Additional version information

This crash seem to have been around for a long time and is not recently introduced.

**Comment by ge-af** [comment]
Hi @DagAgren ,

Thank you for reaching out to us!

To provide optimal support, we would kindly ask you to submit a support ticket to support@appsflyer.com.
When submitting the ticket, please specify your AppsFlyer sign-up (account) email, app ID, reproduction steps, code snippets, logs, the links you used, and any additional relevant information.

**Comment by DagAgren** [comment]
It seems this crash was introduced in December 2020 when we changed from using the Cocoapod "Beta-AppsFlyerFramework" version 6.0.1.168, to "AppsFlyerFramework" version 6.1.0.

**Comment by ge-af** [comment]
Fix for this one was just released in v6.5.2 of the iOS SDK.
